packs/equipment-effects/effect-rainbow-vinegar-greater.json

```
@@ -1,6 +1,6 @@
 {
     "_id": "CGkWnH402Q45HEVt",
-    "img": "systems/pf2e/icons/default-icons/effect.svg",
+    "img": "systems/pf2e/icons/equipment/consumables/potions/serum-of-sex-shift.webp",
```

```suggestion
    "img": "icons/consumables/potions/potion-flask-corked-green.webp",
```

A little iffy on using this icon here since it is made for the specific one. Besides, the item description says it looks like an oil spill.